### PR TITLE
Ranking i18n

### PR DESCRIPTION
**Pull request by hanlu09205**
Verified with screenshot https://screenshot.googleplex.com/8oP5wkMEyCMWKVf.

### CODE REVIEW

File: static/js/chart/types.ts [hunk 1]
```diff
@@ -89,6 +89,7 @@ export interface PageData {
   nearbyPlaces: string[];
   categories: { string: string };
   names: { string: string };
+  locale: string[];
```
Comment (beets) [inline static/js/chart/types.ts:92]: It doesn't look like this is used (and it shouldn't be necessary). Please revert this change.

**Review by beets — APPROVED** [review]
thanks! lgtm except for one extra locale prop.

File: static/js/chart/types.ts (hunk 1 above)
Comment (hanlu09205) [reply static/js/chart/types.ts]: Thanks for the review. Done.
